js/main.js

```diff
@@ -0,0 +1,21 @@
+//Функция, возвращающая случайное целое число из переданного диапазона включительно
+//Использованы данные с сайта https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Math/random
+// https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Statements/throw
+
+function getRandomInt(min, max) {
+  if (min < 0) {throw 'Incorrect \'min\' value';}
+  min = Math.ceil(min);
```

Comment:
```suggestion
  if (min < 0) {
   throw 'Incorrect \'min\' value';
  }
  
  min = Math.ceil(min);
```

После блока кода пустая строка. 

И блок кода в одну строчку не пишется, это плохая практика